lib/cli.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/purpleidea/mgmt/bindata"
 	"github.com/purpleidea/mgmt/gapi"
+
```

Comment:
What's this?

Reply:
Looks like `gofmt -w lib/cli.go` is fine with no line between that and the comment, but `make gofmt` adds it in.

Reply:
Not in my version of golang. They've been known to break compatibility before. For now we're on 1.10.x. Please remove this one-line nl addition. If there's a change in `gofmt` we can address that in a separate commit. We'll be moving to 1.11.x soon anyways. (We basically track the version in Fedora stable -1.)